src/test/py/artcollection/art_collection_test.py

```diff
@@ -0,0 +1,45 @@
+"""Test module for ArtCollection class"""
```

Comment:
Opportunistic Improvement

This file should have a _shebang_ line here
`#!/usr/bin/env python3`

since it has
```python
if __name__ == "__main__":
    ...
```